components/change_url_modal/index.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
+
+import {getSiteURL} from 'utils/url';
+
+import ChangeURLModal from './change_url_modal';
+
+function mapStateToProps(state, ownProps) {
+    const currentTeam = getCurrentTeam(state);
+    const currentTeamURL = `${getSiteURL()}/${currentTeam.name}`;
+    return {
+        ...ownProps,
```

Comment:
Turns out we don't actually need to do`...ownProps` here (I know we're doing it everywhere else), it's handled automatically. Mind just removing this line?

Reply:
@jwilander I removed this here: https://github.com/mattermost/mattermost-webapp/pull/464/commits/16139e8d7462077e18c1bed97bc9ec13f9465cd8 and also removed `ownProps` as a param.